src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -184,8 +216,8 @@ export class TransactionHelper implements OnModuleInit {
     userData: UserData,
     direction: FeeDirectionType,
     asset: Asset,
-    txAsset: Asset | Fiat,
     txVolume: number,
+    txAsset: Asset | Fiat,
     paymentMethod: BuyPaymentMethod,
```

Comment:
Check parameters. Some can be undefined and should have `T | undefined` (e.g. userData)